SUPPORTED_LANGUAGES.md

```diff
@@ -41,6 +41,7 @@ The table below shows the full list of languages (and corresponding classes/alia
 | C/AL                    | cal                    |         |
 | Cache Object Script     | cos, cls               |         |
 | CMake                   | cmake, cmake.in        |         |
+| COBOL                   | cobol, standard-cobol   | [highlightjs-cobol](https://github.com/otterkit/highlightjs-cobol) |
```

Comment:
Aliases like this only exist for user convenience... is there a reason a user would choose to [prefer and] write out the much longer `standard-cobol` when they could just write `cobol`?

Reply:
This could be useful if a COBOL vendor decides to make their own highlightjs package later on.

COBOL has many variants and dialects, and it could be useful in the future to differentiate between Standard COBOL, Enterprise COBOL, Visual COBOL, etc.

Reply:
A user might prefer to use the `standard-cobol` alias when using multiple dialects on the same page. This is due to incompatibilities between variants.

Reply:
Aliases as future proofing... interesting.  I'm not sure anyone else is doing that, but ok. :)